pybotvac/exceptions.py

```diff
@@ -0,0 +1,11 @@
+class NeatoException(Exception):

```

Comment:
I think we should add some short comments to describe every type of exception and the reason why it could be raised. This way we'll get a good feeling if this type is really necessary.

Reply:
ok will add some descriptions.